### PR TITLE
Issue #SB-7140 feat: telemetry interact data for left and right arrow…

### PR DESCRIPTION
**Pull request by revathitarento**
… click

### CODE REVIEW

File: src/app/client/src/app/modules/home/component/main-home/main-home.component.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { CoursesService, UserService, PlayerService } from '@sunbird/core';
 import { ResourceService, ToasterService, ServerResponse, ConfigService, UtilService} from '@sunbird/shared';
 import {  IInteractEventObject, IInteractEventEdata, IImpressionEventInput } from '@sunbird/telemetry';
 import * as _ from 'lodash';
+import * as $ from 'jquery';
```
Comment (rajeevsathish) [inline src/app/client/src/app/modules/home/component/main-home/main-home.component.ts]: please justify the use of Jquery in angular code

File: src/app/client/src/app/modules/shared/components/page-section/page-section.component.ts [hunk 2]
```diff
@@ -160,6 +162,14 @@ export class PageSectionComponent implements OnInit {
     if (visits.length > 0) {
       this.visits.emit(visits);
     }
+    const $next = $('.slick-slider').find('.slick-next');
```
Comment (rajeevsathish) [inline src/app/client/src/app/modules/shared/components/page-section/page-section.component.ts]: why have we used Jquery code can't we handle this using angular

Comment (revathitarento) [reply src/app/client/src/app/modules/shared/components/page-section/page-section.component.ts]: Removed jquery, used the ngx core function to detect the click

File: src/app/client/src/app/modules/home/component/main-home/main-home.component.ts (hunk 1 above)
Comment (revathitarento) [reply src/app/client/src/app/modules/home/component/main-home/main-home.component.ts]: removed

**Pull request merged**